frontends/web/src/routes/settings/components/device-settings/bip85-setting.tsx

```diff
@@ -0,0 +1,113 @@
+/**
+ * Copyright 2023 Shift Crypto AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { SettingsItem } from '../settingsItem/settingsItem';
+import { View, ViewButtons, ViewContent, ViewHeader } from '../../../../components/view/view';
+import { Button } from '../../../../components/forms';
+import { ChevronRightDark, PointToBitBox02, Warning } from '../../../../components/icon';
+import { invokeBIP85 } from '../../../../api/bitbox02';
+
+type TProps = {
+  deviceID: string;
+  canBip85: boolean;
+}
+
+export const Bip85Setting = ({ deviceID, canBip85 }: TProps) => {
+  const { t } = useTranslation();
+  const [status, setStatus] = useState<'idle' | 'info' | 'progress'>('idle');
+
+  if (canBip85) {
+    return (
+      <SettingsItem
+        settingName={t('deviceSettings.expert.showDeterministicEntropy.title')}
+        secondaryText={t('deviceSettings.expert.showDeterministicEntropy.description')}
+        extraComponent={<Warning width={20} height={20} />}
+        displayedValue="Requires newer firmware"
```

Comment:
how about `t('bitbox02Wizard.outOfDate'`)?

Reply:
I changed to bitbox02Wizard.advanced.outOfDate for now. it s a bit long :)

but the content is good imo. 👍 

It's a bit broken on mobile as the text doesn't wrap nicely.

![Screen Shot 2023-12-28 at 17 48 08](https://github.com/digitalbitbox/bitbox-wallet-app/assets/546900/ef2870d0-b295-423c-a83b-878cdc77a76c)